rust/src/private.rs

```diff
@@ -98,4 +98,37 @@ pub fn get_account_balance() -> Result<(), Box<dyn Error>> {
         },
     };
     Ok(())
+}
+
+pub async fn get_account_balance_async() -> Result<(), Box<dyn Error>> {
+    let payload = "\"\\\"\\\"\"";
+    let nonce = get_nonce();
+    let checksum = get_checksum(&payload);
+    let nonce_checksum = get_nonce_checksum(&nonce.to_string(), &checksum);
+    let signature = get_signature(config::get_private_api_endpoint(&"ACCOUNT_BALANCE"), &nonce_checksum);
+    let (secret_key, api_key) = credential(&env!("SECRET_KEY"), &env!("API_KEY"));
+    let param = ["?nonce=", &nonce.to_string(), "&checksum=", &checksum, "&payload=", ""].concat();
+    let client = reqwest::Client::builder()
+        .timeout(Duration::from_secs(config::timeout()))
+        .build()?;
+    let temp = URLBuilder.format (
+        config::url_api_bitwyre(),
+        config::get_private_api_endpoint(&"ACCOUNT_BALANCE"),
+        &param
+    );
+    let res = client.get(&temp)
+        .header("API_Key", api_key)
+        .header("API_Sign", signature)
```

Comment:
this one should be "API-Key" & "API-Sign" otherwise it will produce a 422 error just like in my PR before this

Reply:
changed, please check again.